### ISSUE TITLE
Screenshots

### ISSUE THREAD

**Issue by btd1337** [locness3/kanboard-modern-CSS#1]
Please, Could you to add some screenshoots in your readme?

**Comment by moeenio** [comment]
Yes, I may do it but keep in mind this is in a very early stage and there aren't much big changes.

**Comment by moeenio** [comment]
Hey @btd1337, I made many updates and added a screenshot.

**Comment by ghost** [comment]
Maybe more screenshots would be nice.

**Comment by moeenio** [comment]
ikr, but there isn't a lot of areas the theme styles

**Comment by ghost** [comment]
Hm.. Yeah.
Is your kanboard using Modern Theme?

**Comment by moeenio** [comment]
It is

Also, please don't make links to my server like this :P

**Comment by ghost** [comment]
Okay, sorry.

**Comment by moeenio** [comment]
You see, github comments have a history 🙃 

Fine, though, i'm not blaming you :P

**Comment by ghost** [comment]
Yes.
![image](https://user-images.githubusercontent.com/45879791/84591124-288fb980-ae3c-11ea-94ef-caeee351e631.png)

**Comment by moeenio** [comment]
That applies to yours (this thread has really gone offtopic lol)

**Comment by ghost** [comment]
I know.
